tasks/sign-index-image/sign-index-image.yaml

```
@@ -63,6 +66,7 @@ spec:
         else
           manifest_digest="${reference_image#*@}"
         fi
+        plr_key="internal-services.appstudio.openshift.io/pipelinerun-uid"
```

I hate abbreviations :upside_down_face: 
```suggestion
        pipelinerun_label="internal-services.appstudio.openshift.io/pipelinerun-uid"
```

---

Sure thing. Updated